plugins/plugin-svelte/test/plugin.test.js

```diff
@@ -0,0 +1,69 @@
+const mockCompiler = jest.fn().mockImplementation(() => ({js: ''}));
+const mockPreprocessor = jest.fn();
+
+const path = require('path');
+
+const mockConfig = {buildOptions: {sourceMaps: false}, installOptions: {rollup: {plugins: []}}};
+const mockComponent = path.join(__dirname, 'Button.svelte');
+
+describe('@snowpack/plugin-svelte', () => {
+  describe('mocked ', () => {
+    let plugin;
+
+    beforeAll(() => {
+      jest.mock('svelte/compiler', () => ({compile: mockCompiler, preprocess: mockPreprocessor})); // important: mock before import
+      plugin = require('../plugin');
+    });
+
+    afterAll(() => {
+      jest.unmock('svelte/compiler');
+    });
+
+    afterEach(() => {
+      mockCompiler.mockClear();
+      mockPreprocessor.mockClear();
+    });
+
+    it('passes options to compiler', () => {
+      const options = {
+        generate: 'ssr',
```

Comment:
This is the main bug we’re testing for: `generate: 'ssr'` wasn‘t being passed to the plugin. This test failed before the change.